packages/design-system/src/icons/index.js

```diff
@@ -132,3 +132,6 @@ export { default as TrashFilledSmall } from './trash_filled_small.svg';
 export { default as TurningLine } from './turning_line.svg';
 export { default as Union } from './union.svg';
 export { default as Video } from './video.svg';
+export { default as ProductCheck } from './product_check.svg';
+export { default as ProductPlus } from './product_plus.svg';
+export { default as ProductRemove } from './product_remove.svg';
```

Comment:
`cross.svg` on a red background with border radius? Then we can avoid this extra icon.

(Note: colors should be defined in the theme wherever possible and not hardcoded in CSS)